bot/exts/moderation/incidents.py

```diff
@@ -394,18 +484,76 @@ async def on_raw_reaction_add(self, payload: discord.RawReactionActionEvent) ->
                 return
 
             if not is_incident(message):
-                log.debug("Ignoring event for a non-incident message")
+                log.debug("Ignoring event for a non-incident message.")
                 return
 
             await self.process_event(str(payload.emoji), message, payload.member)
-            log.trace("Releasing event lock")
+            log.trace("Releasing event lock.")
 
     @Cog.listener()
     async def on_message(self, message: discord.Message) -> None:
         """Pass `message` to `add_signals` if and only if it satisfies `is_incident`."""
         if is_incident(message):
+            message_links = DISCORD_MESSAGE_LINK_RE.findall(str(message.content))
+
+            if message_links:
+                embeds = []
+                for message_link in message_links:
+                    ctx = await self.bot.get_context(message)
+                    embeds.append(await make_message_link_embed(ctx, message_link[0]))
+
+                webhook = await self.bot.fetch_webhook(Webhooks.incidents)
```

Comment:
The webhook should be a cog level attribute to not fetch it every time.

Reply:
Resolved in commit 92f37ec and 0f8ecf3.